CODEOWNERS

```diff
@@ -116,6 +116,8 @@
 /drivers/led_strip/                       @mbolivar
 /drivers/pinmux/stm32/                    @rsalveti @idlethread
 /drivers/sensor/                          @bogdan-davidoaia @MaureenHelm
+/drivers/sensor/ams_iAQcore               @alexanderwachter
+/drivers/sensor/ens210                    @alexanderwachter
```

Comment:
add trailing slash to signify a directory.

Reply:
Thx. Done